Fixes for tests

backsapc opened the pull request:
The behavior changed after https://github.com/warden-protocol/wardenprotocol/pull/1084/files#diff-3399e399c1ab8cf55584e6cf64b6a7a26cf4c60d0500c840ae3e103194f5405b, so I changed test's expectation

backsapc committed:
Fixes for tests

pitasi commented:
can you explain why this test was working before and now it's failing? did the error message change?

backsapc commented:
> can you explain why this test was working before and now it's failing? did the error message change?

Yes, I attached the diff that caused this. Initially, we were returning an error if there were no `signRequests` matching the specified search criteria. This was a mistake in the initial implementation, so I corrected it. However, I didn't expect that we had a test for this case, so I updated the test case to reflect the new behavior.
